gtsfm/utils/geometry_comparisons.py

```diff
@@ -248,3 +248,32 @@ def compute_points_distance_l2(wti1: Optional[Point3], wti2: Optional[Point3]) -
     if wti1 is None or wti2 is None:
         return None
     return np.linalg.norm(wti1 - wti2)
+
+
+def get_points_within_radius_of_cameras(wTi_list: List[Pose3], points_3d: np.ndarray, radius: float = 50) -> Optional[np.ndarray]:
```

Comment:
I guess this will be dependent on each scene? Or is 50 a high enough value that most datasets will fit within it?

Reply:
Technically I think we should be using `react-three-fiber` soon. So I think this will be a moot point then, and can be removed.

But a 50 m radius improved the plots a lot for now, and seems to help with debugging.